src/wp-includes/class-wp-theme.php

```diff
@@ -824,6 +824,38 @@ public function cache_delete() {
 		$this->delete_pattern_cache();
 	}
 
+	/**
+	 * Get block pattern cache.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return mixed Returns an array of patterns if cache is found, otherwise false.
+	 */
+	public function get_pattern_cache() {
+		if ( ! $this->exists() ) {
+			return false;
+		}
+		$pattern_data = get_transient( 'wp_theme_patterns_' . $this->stylesheet );
+		if ( is_array( $pattern_data ) && $pattern_data['version'] === $this->get( 'Version' ) ) {
+			return $pattern_data['patterns'];
+		}
+		return false;
+	}
+
+	/**
+	 * Set block pattern cache.
+	 *
+	 * @since 6.4.0
+	 */
+	public function set_pattern_cache( array $patterns = array() ) {
```

Comment:
Why should this have a default? I don't think that makes sense, as it makes you think the parameter is optional, but it shouldn't be. If the parameter is optional, it could suggest that the method can also take care of figuring out the patterns itself, which is not the case.
```suggestion
	public function set_pattern_cache( array $patterns ) {
```